File: polyglot-web/src/main/java/be/artisjaap/polyglot/web/endpoints/response/TranslationsLoadedByFileResponse.java
package be.artisjaap.polyglot.web.endpoints.response;

import lombok.*;

import java.util.ArrayList;
import java.util.List;


@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class TranslationsLoadedByFileResponse {
    @NonNull
    private String languagePairId;
    private LessonHeaderResponse lessonHeaderResponse;
    @Builder.Default
    private List<TranslationResponse> translations = new ArrayList<>();
}

File: polyglot-web/src/main/front-modules/polyglot-R/src/app/home/model/translations-loaded-by-file-response.ts
import {TranslationForLessonResponse} from './translation-for-lesson-response';
import {LessonHeaderResponse} from './lesson-header-response';

export class TranslationsLoadedByFileResponse {
  lessonHeaderResponse: LessonHeaderResponse;
  languagePairId: string;
  translation: TranslationForLessonResponse;

}
